misc/enr/src/lib.rs

```diff
@@ -575,14 +661,21 @@ impl Decodable for Enr {
         seq[8 - seq_bytes.len()..].copy_from_slice(&seq_bytes);
         let seq = u64::from_be_bytes(seq);
 
-        let mut content = IndexMap::new();
+        let mut content = BTreeMap::new();
+        let mut keys = Vec::new();
```

Comment:
Nice!. One final change to merge would be to modify this to something like:
```rust
let mut prev_key: Option<String> = None;
for _ in 0..decoded_list.len() /2 {
    let key = decoded_list.remove(0);
    ....
    if prev_key.is_some() && prev_key >= key {
        return Err(DeocderError::Custom("Unsorted keys"));
     } 
     ...
```

Reply:
Was doing something similar without the `Option`  type which looked very convoluted.
This is great, thanks!